pkg/internal/resources/util/labels.go

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2019 The KubeCarrier Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+const (
+	// Kubernetes recommended labels labels: https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels
```

Comment:
Let's not bloat `util` package even more, please place these labels somewhere else, maybe even have `labels` package which shall store all these string label constants?

Reply:
Overall great PR, just address this nit and we're good to go. 

(for acceptance just move all labels somewhere not in `util`, other label refactoring is another isssue/PR)

Reply:
I prefer to keep the label as close as possible to the usage of it, this util folder is created in this PR. I don't really see the benefits to keep all labels among different controllers/packages in a cental place.

Reply:
`util` is a generic, catch-all name; see https://blog.golang.org/package-names

At least for now move the resource-specific labels to the resource package, e.g. `/pkg/internal/resources/constants` or something like that.

Let's keep the `util` package as lean as possible if we can help it.